src/promptflow-evals/promptflow/evals/evaluators/_chat/_chat.py

```diff
@@ -22,39 +22,41 @@
 
 
 class ChatEvaluator:
+    """
+    Initialize an evaluator configured for a specific Azure OpenAI model.
```

Comment:
Could you please make the class descriptions for all evaluators more specific? This will improve the documentation's clarity.

A few examples:
1. ChatEvaluator:  Initialize a chat evaluator configured for a specific Azure OpenAI model.
2. CoherenceEvaluator:  Initialize a coherence evaluator configured for a specific Azure OpenAI model.
3. ContentSafetyChatEvaluator: Initialize a content safety evaluator to evaluate content safetry metrics for Chat scenario.

Reply:
Fixed.